### DISCUSSION TITLE
Logout button is not working like it was.

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#383] [General]
Hello,

Please note that the "Logout" button is not working properly like it was; it just change the username label to "Login" and screen is not transferred to the login page. you can keep browsing after hitting the Logout Button, the connection is still established.

Thanks.

**Comment by rejetto** [comment]
i made some tests and cannot see any problem.
1. i access a protected folder
2. i'm asked for login
3. i enter credentials
4. i see the folder content
5. i click logout
6. i'm asked for login again

please double check, and provide more information, possibly a video with config (at least the vfs part)

**Reply by GeorgeGeorgeGe** [reply]
I am running HFS as a service, after receiving your reply i restarted the service and it worked fine.
Sorry for that.
thanks for your quick reply.
Regards

**Reply by GeorgeGeorgeGe** [reply]
Hello,
Just to clarify my first message in this discussion:
when you sign in with the below popup screen, you get the behaviour i mentioned in my first message:
![image](https://github.com/rejetto/hfs/assets/130875039/66e176e5-f8a7-4792-92a6-7a4ed514072e)

but when you click cancel and sign in in the page below, things work properly:
![image](https://github.com/rejetto/hfs/assets/130875039/3ea339a9-2ddc-4dc3-9844-9e4a2fd98ed6)

Regards

**Comment by rejetto** [comment]
you are expected to get the first dialog only if you are linking a file.
Are you linking a file?

**Comment by GeorgeGeorgeGe** [comment]
Would you please tell me what do you mean by "linking a file" ?
the first dialogue above is the first dialogue the user gets when he tries to access my web shares:
https://[my web site]:[port number]

Regards

**Comment by rejetto** [comment]
that's strange, if i try to access an hfs from the root, i get the second dialog, not the first one.
<img width="241" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/5d429476-d12d-470b-9426-7105f69e631b">

Please double check.
Can you provide a video?

linking a file means that the address in the browser is pointing to a file instead of a folder

**Comment by rejetto** [comment]
ok maybe it's because you have an index.html on the root.
Let me think.

**Comment by GeorgeGeorgeGe** [comment]
Ok, i will provide a video in a few hours.
Thanks for your quick support.


Sent from Yahoo Mail for iPhone


On Sunday, October 22, 2023, 2:11 PM, Massimo Melina ***@***.***> wrote:




that's strange, if i try to access an hfs from the root, i get the second dialog, not the first one.


Please double check.
Can you provide a video?

**Comment by rejetto** [comment]
don't need the video anymore, i've found how to reproduce the problem

**Comment by GeorgeGeorgeGe** [comment]
No i do not have html file on root.The root is a folder on the physical PC.


Sent from Yahoo Mail for iPhone


On Sunday, October 22, 2023, 2:14 PM, Massimo Melina ***@***.***> wrote:




ok maybe it's because you have an index.html on the root.
Let me think.

**Comment by rejetto** [comment]
are you totally sure that this is not enabled?
<img width="351" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/7d5b157a-382b-4e04-b614-ac3bd0bf5aeb">

**Reply by GeorgeGeorgeGe** [reply]
I have just checked. it is NOT enabled.

**Comment by rejetto** [comment]
then i have no idea.
would you give me your config?
admin > options > edit config >  copy

**Reply by GeorgeGeorgeGe** [reply]
Ok, give me a few hours to re-configure the shares from scratch, if it does not work, i will send you the config file.

**Reply by rejetto** [reply]
if you want to change the names manually, do it.
That's not important to me

**Reply by GeorgeGeorgeGe** [reply]
I have more than 50 Users, 10 users group and 20 shared folders.
it is not easy to change names manually.
give me some time please.
Thanks

**Reply by rejetto** [reply]
take your time, and if you prefer send it privately: a@rejetto.com

**Comment by GeorgeGeorgeGe** [comment]
I installed previous version (ver. 0.49.0-alpha4) and it worked perfect.
i re-installed ver. 0.49.0-alpha5, it did not work ( same behaviour as in my first message).
Please check.

Regards

**Comment by rejetto** [comment]
i studied all changes of alpha4-5 and found nothing useful.
But I've fixed a problem that seems related,
so would you please tell me if this version fixes your problem? [hfs-windows.zip](https://github.com/rejetto/hfs/files/13114777/hfs-windows.zip)
thanks

**Reply by GeorgeGeorgeGe** [reply]
Yes, That fixed the problem, Perfect !
Thanks for your great support !
